Properties/serviceDependencies.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
what's the usage of this and the other file (serviceDependencies.local.json)? 

Note: use upper case for the first letter in the first word in file names.